master/internal/resourcemanagers/kubernetes/pods.go

```diff
@@ -370,12 +370,13 @@ func (p *pods) receiveJobQueueMsg(ctx *actor.Context) {
 					"received change position command for unregistered container id")
 				return
 			}
+
 			// check that the pod exists
 			_, err := p.clientSet.CoreV1().Pods("default").Get(context.TODO(), podName, metaV1.GetOptions{})
 			if err != nil {
 				ctx.Log().WithField("pod-id", msg.PodID).Info(
 					"change position command failed with err: ", err)
-			}
+      }
```

Comment:
nit: is this intentional/correct? ci go fmt set up seems to not be catchin go formatting issues in generanl